http-api/src/v1/projects.rs

```diff
@@ -791,11 +763,10 @@ fn resolve_revisions(
                 // Iterates over commits headers and retrieves each commit details until it gets to the head of the default branch
                 // If radicle_source::commit returns a None the commit won't be collected.
                 let commits = commits
-                    .headers
                     .iter()
```

Comment:
Same again, you might be able to `into_iter` here.

Reply:
This would be nice, but we would need to implement `IntoIter` for `History` to be able to do this.

Reply:
Will do a comment somewhere on `radicle_git` to take that into account on the redesign